graylog2-server/src/test/java/org/graylog2/plugin/MessageTest.java

```diff
@@ -772,15 +772,7 @@ public void testTimestampNoConversionWithNullDate() {
     @Test
     public void testNullDateGetsReplacesWithCurrentDate() {
         final Message message = new Message("message", "source", null);
-
         assertThat(message.getTimestamp()).isInstanceOf(DateTime.class);
-
-        assertThat(message.processingErrors()).satisfies(e -> {
```

Comment:
Why do these tests have to go, aren't they all concerned with a missing timestamp and not the grace period handling?

Reply:
Because we are no longer generating processing errors when the timestamp is adjusted; including null replacement.